openstudiocore/src/utilities/core/Path.hpp

```diff
@@ -22,11 +22,12 @@
 
```

Comment:
OK i think the same comment applies. Can we do some consolidation here? Path, PathHelpers, Tempdir... I would like one place (namespace, source code directory) where all file related things are at, with no duplicity or confusion. I think I'm going to have to review this body of code outside of the PR. It is hard to see the big picture from the PR diff. BUT if there is consolidation and cleanup that needs to happen I do not think it needs to be part of this PR or @lefticus task.